### PR TITLE
Fix usability issues / DDR changes

### PR DESCRIPTION
**Pull request by chatterjeeshekhar**

### CODE REVIEW

File: templates/email_body.html [hunk 1]
```diff
@@ -1,5 +1,130 @@
...
+                                            <td>
+                                                <a href="https://u12056031.ct.sendgrid.net/ls/click?upn=r6zPJZULjp56Ste1olcOddoWuFoDqz3YqO8zApfXkhYp6IGy0dbI9ntlJi-2BWZK4usHpP_l-2FPW-2FNsVkcneFy4FpbjnphlzLC-2F8-2FTHRWmB4UlS7eatiA7NyDWPJhA4eVAXxEwSA7uM6DKZjQnxsZiyeCKAjB6cT0qwPbUXel7-2FhXqtxn4gcV-2BGfa4-2Bwopcn6dI7BCxYl1t29IRG9ioJAd-2BE5A3UrkeBlZy5ojpgD7QsME-2FXiUZxmpLea22BR5Y-2F-2FxhbaqcBMPXnXmsoSXH0QTV9T1LevIIQw9dOeZMPb244U4LDizQVdgN7Hi3JJiHwSBFK7fyPQ0IWKpxyaeo5M1reT7QucCCStFjQuemIl6fu0Z-2FOiCg-3D" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://u12056031.ct.sendgrid.net/ls/click?upn%3Dr6zPJZULjp56Ste1olcOddoWuFoDqz3YqO8zApfXkhYp6IGy0dbI9ntlJi-2BWZK4usHpP_l-2FPW-2FNsVkcneFy4FpbjnphlzLC-2F8-2FTHRWmB4UlS7eatiA7NyDWPJhA4eVAXxEwSA7uM6DKZjQnxsZiyeCKAjB6cT0qwPbUXel7-2FhXqtxn4gcV-2BGfa4-2Bwopcn6dI7BCxYl1t29IRG9ioJAd-2BE5A3UrkeBlZy5ojpgD7QsME-2FXiUZxmpLea22BR5Y-2F-2FxhbaqcBMPXnXmsoSXH0QTV9T1LevIIQw9dOeZMPb244U4LDizQVdgN7Hi3JJiHwSBFK7fyPQ0IWKpxyaeo5M1reT7QucCCStFjQuemIl6fu0Z-2FOiCg-3D&amp;source=gmail&amp;ust=1592380950223000&amp;usg=AFQjCNHmgLVNMXK3M8GxAoKRl7GRgCroMA">
```
Comment (Shwetabhk) [inline templates/email_body.html:26]: all of the links in this have sendgrid url
open each. of these urls and see where they're redirecting and add those urls instead
also take these urls from environment variables

File: templates/email_body.html [hunk 2]
```diff
@@ -1,5 +1,130 @@
...
+                                                <img height="100" src="https://ci5.googleusercontent.com/proxy/lplPQC-OzTNA5zv4PyVm8KEcuI6DVbVvrG2LVDGYAjcchVPxNx5HkxV7yOCHiOhjiajTvEJ5hSDo4H7aSE2Z7J16aHIeqNPrZ_00-S-qArxlq-wkH2d8GIdQC8o=s0-d-e1-ft#https://app.fyle.in/app/main/assets/images/email_templates/pdf_export.png" alt="PDF Report" class="CToWUd">
+                                                </a>
+                                            </td>
+                                        </tr>
+                                        <tr align="center">
+                                            <td style="font-size:20px;font-weight:bold">
+                                                Export is ready for download
```
Comment (Shwetabhk) [inline templates/email_body.html:33]: Backup is ready for download

File: templates/email_body.html [hunk 3]
```diff
@@ -1,5 +1,130 @@
...
+                                                Export is ready for download 
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                <table style="padding:20px 0px 20px 0px">
+                                                    <tbody>
+                                                        <tr>
+                                                            <td style="font-size:14px;color:#4a4a4a">
+                                                                Please click on the button below to download the export you requested. The download link will expire in 7 days calendar days from now.
```
Comment (Shwetabhk) [inline templates/email_body.html:42]: backup instead of export

File: templates/email_body.html (hunk 1 above)
Comment (chatterjeeshekhar) [reply templates/email_body.html:26]: It happens to redirect to https://app.fyle.in/app/accounts/#/signin?

**Pull request merged**